svelte/src/scripts/create-graph-data/index.ts

```diff
@@ -21,17 +21,17 @@ function assignParentReference(nodes: GraphDataNode[]) {
 	});
 }
 
-function assignOutgoingAndIncomingLinksAndOriginalSourceAndTargetReference(
+function assignOutgoingIncomingLinksAndOriginalLiftedSourceTargetReference(
```

Comment:
What a name.

You can just call it something like assignLinkReferences and use a jsdoc comment to explain what exectly it does.

Reply:
agree :v